tests/test_wasm_build.py

```
@@ -0,0 +1,290 @@
+            "init",
+        ]
+
+        for method in required_methods:
+            assert method in content, f"Missing method: {method}"
+
+
+class TestBuildTools:
+    """Tests for build tool detection."""
+
+    @pytest.mark.skipif(
+        subprocess.run(["which", "emcc"], capture_output=True).returncode != 0,
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)